### ISSUE TITLE
Dedicated server can't bind to port

### ISSUE THREAD

**Issue by ClayTeck** [Regalis11/Barotrauma#8563]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
So, I wondered why some servers have ping "?" and discovered that the servers with the valid ping can be pinged through Windows cmd /ping 0.0.0.0 , and "?"-pinged server timing out cmd /ping 0.0.0.0

Later, i found out that i can't bind my dedicated server to ports. I mean, my server is listed, visible by other players, but ports are shown as "closed" and other programs can bind to them like they're not busy. 
It looks like some servers can bind to ports, but some servers can't.
Maybe it's not a glitch, if so i'd like to know how can i fix that. But it looks like a glitch, very common glitch, many servers have "?" ping

**Steps To Reproduce**
1. Host a DedicatedServer.exe
2. Check your ports
3. It's closed
4. If it's not closed, check out other servers in the serverlist
5. Find one dedicated server with valid ping and one with "?" ping
6. Try to ping valid-pinged server, and "?"-pinged server.
7. Use your monitor to see difference between them

**Version**
Latest Stable

**Additional information**
 None

**Comment by cshingle** [comment]
@ClayTeck @ClayTeck I have a few comments and questions.
1) How are you checking for listening ports. There are multiple ways to do this.
2) Can you attach a text file of your dedicated server console output?

Are you using the standard Windows Ping command or something else? The Windows command sends an ICMP packet which is on a different network layer and is not useful for testing for open ports. https://en.wikipedia.org/wiki/Ping_(networking_utility)

It also looks like you are trying to connect to 0.0.0.0 which doesn't make sense it is a gateway, not a valid IP address. You want to try to connect to the IP Address of the system that the dedicated server is running on. If it is running on your local machine you should be able to find it on 127.0.0.1.

What ports are you looking for? I started a dedicated server on my system and this is what I get
`u_str	LISTEN	0	255	/tmp/dotnet-diagnostic-45538-2994695-socket	525808	*	0	users:(("DedicatedServer",pid=45538,fd=10))	
u_dgr	UNCONN	0	0	*	524675	*	524676	users:(("DedicatedServer",pid=45538,fd=88))	
u_dgr	UNCONN	0	0	*	524676	*	524675	users:(("DedicatedServer",pid=45538,fd=90))	
udp	UNCONN	0	0	*:27015	*:*	users:(("DedicatedServer",pid=45538,fd=75))`
Basically this is looking on ports 525808, 524676, 524675, 27015. Note that I am on Linux that has a very different network stack than Windows.
